harvard-lts/drs-translation-service
# Etd 144

**DTS refactor.**
* * *

**GitHub Issue**: [(link)](https://jira.huit.harvard.edu/browse/ETD-144)

* Subtask https://jira.huit.harvard.edu/browse/ETD-146

# What does this Pull Request do?
This PR includes a refactor of DTS translation service, including subclasses for epadd and dataverse.

# How should this be tested?

- Check out this branch from github
- copy env.template to .env and fill with dev values
- bring up docker > docker-compose -f docker-compose-local.yml up --build -d --force-recreate --remove-orphans
 - exec into docker > docker exec -it dts bash
 - run "pytest" tests should pass
- deploy to dev and push batch, should be successful

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests? Y

# Interested parties
Tag (@ mention) interested parties
